src/lib/parse/CompoundValueNode.cpp

```diff
@@ -883,6 +900,18 @@ std::string CompoundValueNode::toJson(bool isLastElement, bool comma)
       out = JSON_STR(key) + ":" + "null";
     }
   }
+  else if (valueType == orion::ValueTypeNotGiven)
+  {
+     LM_E(("Runtime Error (ValueTypeNotGiven (%s))", name.c_str()));
```

Comment:
indent

Reply:
Fixed in e85892cdb28f2f510fd2b0a38cacbb43082f9613